demo/index.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport"
+          content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>spine-context-menu demo</title>
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
+    <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+    <link rel="import" href="../spine-context-menu.html">
+
+    <custom-style>
+        <style is="custom-style" include="demo-pages-shared-styles">
+        </style>
+    </custom-style>
+</head>
+<body>
+<div class="vertical-section-container centered">
+    <h3>Basic spine-context-menu demo</h3>
+    <demo-snippet>
+        <template>
+            <spine-context-menu>
+                <template>
+                    <div style="width: 200px; height: 200px; background: rebeccapurple;
+                        line-height: 200px; text-align: center; color: white;">Dropdown content
+                    </div>
+                </template>
+                <div style="width: 100%; height: 500px; background: cyan;
+                    line-height: 500px; text-align: center; color: white;">Right click me
+                </div>
+            </spine-context-menu>
+        </template>
+    </demo-snippet>
+</div>
+
+<div class="vertical-section-container centered">
+    <h3>Target spine-context-menu demo</h3>
+    <demo-snippet>
```

Comment:
These two demo cases are not covering all the unique features of the component. Please add more demo cases.